workflows/argo-events/serviceaccounts/k8s-secret-events-nautobot.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: argo-events
+  name: k8s-events-secret-nautobot
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
This doesn't need to be a cluster role but can just be singular namespace scoped Role